### ISSUE TITLE
User should be able to copy messages on mobile

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1086]
I think long press can simply copy the message text to clipboard and show a message that this has happened. Or we could show a "copy" button on long press, followed by some indication that text has been copied. This should be as minimal as possible.

**Comment by rajdip-b** [comment]
Hey @holmesworcester, I feel I can do this right away. Should I submit a PR?

**Comment by holmesworcester** [comment]
Yes please!

**Comment by sanjai0py** [comment]
is this issue still open?

**Comment by holmesworcester** [comment]
Yes, I think so! @rajdip-b you didn't get set up on mobile yet did you? Or did you?

See packages/mobile/README.md for instructions on how to get set up on Android! 

If mobile setup gives you trouble, pick another "good first issue" on desktop here: https://github.com/orgs/TryQuiet/projects/3/views/1?filterQuery=label%3A%22good+first+issue%22

Here's the desktop issue for this feature: https://github.com/TryQuiet/quiet/issues/568

**Comment by rajdip-b** [comment]
@holmesworcester just was about to say that! @sanjai0py I faced some dependency error the last time(a few weeks ago) I tried to implement this feature. It might be resolved by now. Perhaps @siepra might enlighten you?

**Comment by holmesworcester** [comment]
Yeah or start with the desktop version of this issue.
